Analysis/Tasks/SkimmingTutorials/DataModel/LFDerived.h

```diff
@@ -15,23 +15,41 @@
 
 namespace o2::aod
 {
-//DECLARE_SOA_TABLE(LFCollisions, "AOD", "LFCOLLISION", o2::soa::Index<>,
-//               o2::aod::bc::RunNumber, o2::aod::collision::PosZ);
-//using LFCollision = LFCollisions::iterator;
+DECLARE_SOA_TABLE(LFCollisions, "AOD", "LFCOLLISION", o2::soa::Index<>,
+                  o2::aod::collision::PosZ);
+using LFCollision = LFCollisions::iterator;
 
 namespace lftrack
 {
-//DECLARE_SOA_INDEX_COLUMN(LFCollision, lfCollision);
+DECLARE_SOA_INDEX_COLUMN(LFCollision, lfCollision);
 DECLARE_SOA_COLUMN(Pt, pt, float);
 DECLARE_SOA_COLUMN(P, p, float);
 DECLARE_SOA_COLUMN(Eta, eta, float);
+DECLARE_SOA_COLUMN(Phi, phi, float);
 DECLARE_SOA_COLUMN(TPCNSigma, tpcNSigma, float[9]);
+DECLARE_SOA_COLUMN(TOFNSigma, tofNSigma, float[9]);
 } // namespace lftrack
 DECLARE_SOA_TABLE(LFTracks, "AOD", "LFTRACK", o2::soa::Index<>,
-                  //lftrack::LFCollisionId,
                   lftrack::Pt, lftrack::P, lftrack::Eta,
                   lftrack::TPCNSigma);
 using LFTrack = LFTracks::iterator;
+
+DECLARE_SOA_TABLE(LFNucleiTracks, "AOD", "LFNUCLEITRACK", o2::soa::Index<>,
+                  lftrack::LFCollisionId,
+                  lftrack::Pt, lftrack::P,
+                  lftrack::Eta, lftrack::Phi,
+                  lftrack::TPCNSigma, lftrack::TOFNSigma);
+using LFNucleiTrack = LFNucleiTracks::iterator;
 } // namespace o2::aod
 
+enum PIDSpecies { El_PID = 0,
```

Comment:
We shall discuss it is not better to have explicit names in the table and avoid the enum